toolkit/partner-chains-cli/src/create_chain_spec/mod.rs

```diff
@@ -76,6 +80,15 @@ impl<T: CreateChainSpecRuntimeBindings> CreateChainSpecCmd<T> {
 		context.print(&format!("- asset name: {}", config.native_token_asset_name));
 		context.print(&format!("- asset policy ID: {}", config.native_token_policy));
 		context.print(&format!("- illiquid supply address: {}", config.illiquid_supply_address));
+		context.print("Governed Map Configuration:");
+		context.print(&format!(
+			"- validator address: {}",
+			config.governed_map_validator_address.clone().unwrap_or_default()
```

Comment:
So, it will print:
```
- validator address: 
```
it some old pc-chain-config.json is used. Doesn't look pretty, but I don't want to sport very hard for this edge case.